package.json

```diff
@@ -62,13 +63,18 @@
     "coverage": "jest --coverage",
     "clean": "rm -rf build/* node_modules/",
     "clean-install": "npm run clean && npm i",
-    "test": "check-dependencies && npm run lint && npm run flow && npm run jest",
-    "verify-screenshots": "check-dependencies && dockers/Screenshotter/screenshotter.sh --verify",
-    "start": "check-dependencies && webpack-dev-server --hot --config webpack.dev.js",
-    "build": "check-dependencies && rimraf build/* && webpack",
+    "test": "npm run prestart && npm run lint && npm run flow && npm run jest",
+    "screenshots": "npm run prestart && dockers/Screenshotter/screenshotter.sh",
+    "verify-screenshots": "npm run screenshots -- --verify",
+    "prestart": "check-dependencies && cd src && node unicodeMake.js >unicodeSymbols.js",
```

Comment:
As generating `unicodeSymbols.js` is cheap, it does before every tests and builds.